composeApp/src/wasmJsMain/resources/styles.css

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 The Android Open Source Project
```

Comment:
![image](https://github.com/user-attachments/assets/2d4c022a-95bc-4ee0-a083-e247272b13d8)

Reply:
het spijt me :D

Reply:
it was never there